### PR TITLE
Resolves RecursionError in latex representation

### LINKED ISSUE #13559: LaTeX representation raises RecursionError
LaTeX representation of an expression that has division by 1 causes `RecursionError`. SymPy version: 1.1.1.

Code to reproduce the bug:
```python
from sympy import latex
from sympy.parsing.sympy_parser import parse_expr

expr = parse_expr('2/1', evaluate=False)
latex(expr)
```
Output:
```
...

~/.virtualenvs/sympy/lib/python3.5/site-packages/sympy/printing/printer.py in _print(self, expr, *args, **kwargs)
    255                 printmethod = '_print_' + cls.__name__
    256                 if hasattr(self, printmethod):
--> 257                     return getattr(self, printmethod)(expr, *args, **kwargs)
    258             # Unknown object, fall back to the emptyPrinter.
    259             return self.emptyPrinter(expr)

~/.virtualenvs/sympy/lib/python3.5/site-packages/sympy/printing/latex.py in _print_Pow(self, expr)
    466         elif expr.exp.is_Rational and expr.exp.is_negative and expr.base.is_commutative:
    467             # Things like 1/x
--> 468             return self._print_Mul(expr)
    469         else:
    470             if expr.base.is_Function:

~/.virtualenvs/sympy/lib/python3.5/site-packages/sympy/printing/latex.py in _print_Mul(self, expr)
    396             # use the original expression here, since fraction() may have
    397             # altered it when producing numer and denom
--> 398             tex += convert(expr)
    399         else:
    400             snumer = convert(numer)

~/.virtualenvs/sympy/lib/python3.5/site-packages/sympy/printing/latex.py in convert(expr)
    366         def convert(expr):
    367             if not expr.is_Mul:
--> 368                 return str(self._print(expr))
    369             else:
    370                 _tex = last_term_tex = ""

... last 4 frames repeated, from the frame below ...

~/.virtualenvs/sympy/lib/python3.5/site-packages/sympy/printing/printer.py in _print(self, expr, *args, **kwargs)
    255                 printmethod = '_print_' + cls.__name__
    256                 if hasattr(self, printmethod):
--> 257                     return getattr(self, printmethod)(expr, *args, **kwargs)
    258             # Unknown object, fall back to the emptyPrinter.
    259             return self.emptyPrinter(expr)

RecursionError: maximum recursion depth exceeded in comparison
```

### PR DESCRIPTION
**Pull request by ashishkg0022**
fixes #13559. It was due to factor of `1/1`  
```python
expr = parse_expr('5/1', evaluate=False)
expr.args
```
it gives `(5 , 1/1)` , which was not handled properly in latex conversion . So everytime an expression was divided by 1 , it resulted in RecursionError

### CODE REVIEW

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -415,12 +416,9 @@ def convert(expr):
                     _tex += term_tex
                     last_term_tex = term_tex
                 return _tex
+        one_by_one =  parse_expr('1/1', evaluate=False)
```
Comment (ylemkimon) [inline sympy/printing/latex.py]: You can do `Pow(1, -1, evaluate=False)`.

**Comment by ashishkg0022** [comment]
@ylemkimon  is it fine now ?

**Comment by ashishkg0022** [comment]
@jksuom can you please see it?

File: sympy/printing/latex.py [hunk 2]
```diff
@@ -415,12 +416,9 @@ def convert(expr):
                     _tex += term_tex
                     last_term_tex = term_tex
                 return _tex
+        one_by_one =  Pow(1, -1, evaluate=False)
 
-        if denom is S.One:
-            # use the original expression here, since fraction() may have
-            # altered it when producing numer and denom
-            tex += convert(expr)
```
Comment (jksuom) [inline sympy/printing/latex.py]: Is it possible to preserve this line if `Pow(1, -1, evaluate=False) not in expr.args` is added to the above condition?`

**Comment by jksuom** [comment]
Thanks, this looks good.

**Pull request merged**